lib/segment/src/vector_storage/mmap_vectors.rs

```diff
@@ -104,57 +99,6 @@ impl MmapVectors {
         self.deleted.flusher()
     }
 
-    pub fn quantize(
-        &mut self,
-        distance: Distance,
-        data_path: &Path,
-        quantization_config: &QuantizationConfig,
-        max_threads: usize,
-        stopped: &AtomicBool,
-    ) -> OperationResult<()> {
-        // In theory, we can lock deleted flags here, as we assume that it is the hottest data. We
```

Comment:
are those comments completely obsolete or should they be propagated somewhere else?

Reply:
In the end we decided to disable mlocking because it had unexpected side effects. I'm not sure if we'll ever try to implement it again.

I don't think this comment is obsolete so we may put it elsewhere. But since it's unclear if we'll ever cover this again I don't see it as big problem if it is lost in this change.

Reply:
To be honest, I don't have an idea where this comment should be placed now

Reply:
Let's drop it then, I just wanted to make sure this was intentional :+1:

Reply:
Yeah. Good catch!